vignettes/RPyGeo.Rmd

```diff
@@ -124,21 +125,29 @@ If you use RPyGeo in another IDE, the help file is displayed in the default brow
 ArcPy functions can be accessed via the `$` operator.
 You can take advantage of the tab completion feature of RStudio.
 After typing the `$` operator all functions of the ArcPy module are listed (which represents an interactive alternative to `rpygeo_search()`).
-For example, to compute the slope from the digital elevation model we have exported, we can type:
+For example, to compute the slope from the digital elevation model (DEM) we have exported at the beginning of this tutorial, type:
 
 ```{r, eval=FALSE}
 arcpy$Slope_3d(in_raster = "elev.tif", out_raster = "slope.tif")
 ```
 
-This created a file named `slope.tif` in the workspace directory, which in our case is the temporary folder (see above in `rpygeo_build_env()`).
-To do that, run `rpygeo_load("slope.tif")` (see also furhter below). <!-- Marc, is that correct? -->
+Executing the function creates a file named `slope.tif` in the workspace directory, which in our case is the temporary folder (see above in `rpygeo_build_env()`).
+To import the output of the ArcGIS processing into R, run:
+
+<!-- Marc, is that correct? -->
+```{r, eval=FALSE}
+slope <- rpygeo_load("slope.tif")
+```
 
-We can use the pipe operator `%>%` to chain ArcPy function together. 
+This imports the ArcGIS slope processing output into R as a raster object named `slope`.
+<!-- Marc: and is this also possible? arcpy$sa$Slope %>% rpygeo_load() -->
```

Comment:
Marc, is it possible to do: `arcpy$sa$Slope %>% rpygeo_load()`
Sorry, I don't have an ArcGIS installation (hopefully next week).

Reply:
Yes this is possible. `arcpy$sa$Slope` creates a temporary Esri Arc/Info Binary Grid in the scratch workspace. `rpygeo_load()` can read Arc/Info Binary Grids.

Reply:
Ok, thanks!